ogr/services/gitlab/service.py

```diff
@@ -128,3 +128,53 @@ def project_create(
         return GitlabProject(
             repo=repo, namespace=namespace, service=self, gitlab_repo=new_project
         )
+
+    def list_projects(
+        self,
+        namespace: str = None,
+        user: str = None,
+        search_pattern: str = None,
+        language: str = None,
+    ) -> List[GitProject]:
+
+        if namespace:
+            group = self.gitlab_instance.groups.get(namespace)
+            projects = group.projects.list(all=True)
+        if user:
+            u = self.gitlab_instance.users.list(username=user)[0]
+            projects = u.projects.list(all=True)
```

Comment:
Can we have a better name, please?

```suggestion
            user_object = self.gitlab_instance.users.list(username=user)[0]
            projects = user_object.projects.list(all=True)
```

Reply:
You're right. I forgot about the variable names, you told me this one before, my bad.